lib/runtime.js

```diff
@@ -70,18 +69,17 @@ exports.enable = function (Module) {
   // inside loops. The compiler generates these keys automatically (and
   // deterministically) when compiling nested import declarations.
   Mp.importSync = function (id, setters, key) {
-    var module = this;
-    setESModule(module);
+    setESModule(this);
```

Comment:
Since there are no closures now and we've tripped up over the local `module` var before I went ahead and made it just `this` so it's more clear when we're looking at it that it's not the other `module`.